version.properties

```diff
@@ -0,0 +1,4 @@
+#Version of the produced binaries. This file is intended to be checked-in.
+#It will be automatically bumped by release automation.
+#TODO we need to sync it with 'build.sbt'. Can sbt load version from this file?
+version=0.8.1
```

Comment:
Yes. I managed to do this by changing the `build.sbt` file as below

`--- a/build.sbt
+++ b/build.sbt
@@ -1,6 +1,16 @@
+import scala.io.Source
 import scala.language.postfixOps
+import scala.util.Try
 
-val playParSeqVersion = "0.8.1"
+val playParSeqVersion = {
+  val pattern = """^version=(.+)$""".r
+  val source = Source.fromFile("version.properties")
+  val version = Try(source.getLines.collectFirst {
+    case pattern(v) => v
+  }.get)
+  source.close
+  version.get
+}`

Reply:
What this does is basically loading the file and pattern-match each line, then use the first match.

Reply:
Great! I pushed this change to my PR. BTW. you can make commits into this PR and push them upstream to my repo.

Reply:
Really cool!!! I updated the PR with this code snippet.